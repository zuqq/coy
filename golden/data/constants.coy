const NEGATIVE_ONE: i64 = -1;

fn main() -> () {
    println!("NEGATIVE_ONE == {}", NEGATIVE_ONE)
}
