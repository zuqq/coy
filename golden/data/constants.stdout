NEGATIVE_ONE == -1
